backend/routes/web.php

```diff
@@ -11,6 +13,12 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/todos','TodosController@index')->name('todos_index');
+
+Route::get('/todos/create','TodosController@create')->name('todos_create');
+Route::post('/todos/create','TodosController@store')->name('todos_store');
+
+Route::get('/todos/{id}/edit','TodosController@edit')->name('todos_edit');
+Route::post('/todos/{id}/edit','TodosController@update')->name('todos_update');
+
+Route::post('/todos/{id}/delete','TodosController@destroy')->name('todos_delete');
```

Comment:
シンプルなCRUDであれば以下のように書くと自動でいい感じにつくってくれます！

```suggestion
Route::resource('todos', TodosController::class);
```

https://laravel.com/docs/8.x/controllers#resource-controllers

Reply:
ちょっと今回自信がなかったのと、丁寧にやってみました。
今後の参考にしたいと思いますm(__)m